addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
```

Comment:
Strings cannot be compared like this, the enum of the command however can be compared with == please fix this.